src/platform/silabs/EFR32/OTAImageProcessorImpl.cpp

```diff
@@ -162,8 +166,13 @@ void OTAImageProcessorImpl::HandleFinalize(intptr_t context)
             writeBuffer[writeBufOffset] = 0;
             writeBufOffset++;
         }
-
+#if (defined(EFR32MG24) && defined(WF200_WIFI))
+        pre_bootloader_spi_transfer();
```

Comment:
if those can be safely done within bootloader_eraseWriteStorage why surround these here